projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -74,13 +84,15 @@ export class CartItemListComponent {
   constructor(
     activeCartService: ActiveCartService,
     selectiveCartService: SelectiveCartService,
+    userIdService: UserIdService,
```

Comment:
It should not be present in this constructor variant and it should be after `featureConfigService`

Reply:
Also don't forget to update deprecation ts docs